integration/project.yaml

```diff
@@ -5,7 +5,6 @@ test:
     - "VAULT_TOKEN=mytoken"
     - "CIRCLE_SHA1=${CIRCLE_SHA1}"
     - "COMMON_DOMAIN_GUEST=${COMMON_DOMAIN_GUEST}"
-    - "CLUSTER_NAME=${CLUSTER_NAME}"
```

Comment:
This got sorted in the code now. See also https://github.com/giantswarm/aws-operator/pull/831/files#diff-f7ce4f5d19d2fabad8dfe118cbc217dfR31.